vhdbuilder/packer/cleanup.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash -x
+
+required_env_vars=(
+  "CLIENT_ID"
+  "CLIENT_SECRET"
+  "TENANT_ID"
+  "SUBSCRIPTION_ID"
+  "PKR_RG_NAME"
+  "MODE"
+  "AZURE_RESOURCE_GROUP_NAME"
+)
+
+for v in "${required_env_vars[@]}"; do
+  if [ -z "${!v}" ]; then
+    echo "$v was not set!"
+    exit 1
+  fi
+done
+
+#clean up the temporary storage account
+id=$(az storage account show -n ${SA_NAME} -g ${AZURE_RESOURCE_GROUP_NAME} | jq .id)
```

Comment:
Is SA_NAME a required env variable? Didn't see it in the list of "required_env_vars".

Reply:
Done!!